CRM/Prospect/Setup/CreateProspectMenus.php

```diff
@@ -37,44 +41,45 @@ private function createProspectMenuItems() {
       'permission_operator' => 'OR',
       'is_active' => 1,
       'permission' => 'access my cases and activities,access all cases and activities',
-      'icon' => 'crm-i fa-folder-open-o'
+      'icon' => 'crm-i fa-folder-open-o',
     ];
 
     $prospectMenu = civicrm_api3('Navigation', 'create', $params);
-    //Menu weight seems to be ignored on create irrespective of whatever is passed, Civi
-    //will assign the next available weight. This fixes the issue.
+    // Menu weight seems to be ignored on create irrespective of whatever is
+    // passed, Civi will assign the next available weight. This fixes the issue.
     civicrm_api3('Navigation', 'create', [
-        'id' => $prospectMenu['id'],
-        'weight' => $casesWeight + 1]
-    );
+      'id' => $prospectMenu['id'],
+      'weight' => $casesWeight + 1,
+    ]);
     $this->createProspectSubmenus($prospectMenu['id']);
   }
 
   /**
    * Creates Prospect sub menu items.
    *
    * @param int $prospectMenuId
+   *   Prospect menu id.
    */
   private function createProspectSubmenus($prospectMenuId) {
     $submenus = [
       [
         'label' => ts('Dashboard'),
         'name' => 'prospect_dashboard',
-        'url' => 'civicrm/case/a/#/case?category=prospect',
+        'url' => 'civicrm/case/a/#/case?category=prospecting',
         'permission' => 'access my cases and activities,access all cases and activities',
         'permission_operator' => 'OR',
       ],
       [
         'label' => ts('New Prospect'),
         'name' => 'new_prospect',
-        'url' => 'civicrm/case/add?category=prospect',
+        'url' => 'civicrm/case/add?category=prospecting',
```

Comment:
@deb1990 , Can we change this to `civicrm/case/add?category=prospecting&action=add&reset=1`
Adding the reset=1 helps clear the smarty cache for the form so it does not carry values over from it's prior invocation. I see so many references to the action parameter in the Case form BAO too so I think it's a good idea to add that.